src/app/pages/profile/transaction-information/transaction-information.component.spec.ts

```diff
@@ -0,0 +1,150 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpClientModule } from '@angular/common/http';
+import { of } from 'rxjs';
+import { By } from '@angular/platform-browser';
+
+import { AppModule } from './../../../app.module';
```

Comment:
This file could use an absolute imports `make-over` . We can fix all relative paths present in this file

Reply:
I worked on that.. thanks @Tevinthuku